src/Offer/OfferControllerProvider.php

```diff
@@ -69,30 +70,39 @@ function (Application $app) use ($offerType) {
                 }
             );
 
+            /* @deprecated */
```

Comment:
Kan je eventueel de deprecated endpoints groeperen (onderaan deze method bijvoorbeeld), zodat we deze later gemakkelijk tezamen kunnen verwijderen?